12-26 00:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 01:01:00 [INFO] zhenxun | CMD[定时任务] 自动清理临时数据完成，共清理了 15.94MB 的数据...
12-26 01:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 02:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 02:05:45 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.private.friend]: Message 636851579 from 2747277822 '[json:data={"app":"com.tencent.gamecenter.mall","desc":"斗地主秀操作抽188Q币","meta":{...]'
12-26 02:05:45 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 02:05:45 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 02:05:45 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 02:05:45 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 02:05:45 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 02:05:45 [ERROR] nonebot | Running Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) failed.
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x000001E7EB056290>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x000001E7EB0560E0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000001E7EB82BD00>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 579, in run
    server.run()
    │      └ <function Server.run at 0x000001E7EB82B640>
    └ <uvicorn.server.Server object at 0x000001E7FC34C7C0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000001E7EB82B6D0>
           │       │   └ <uvicorn.server.Server object at 0x000001E7FC34C7C0>
           │       └ <function run at 0x000001E7E8D80E50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x000001E7EDB73DF0>
           │    └ <function BaseEventLoop.run_until_complete at 0x000001E7E8E80280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x000001E7EAA91510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x000001E7E8E81CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x000001E7E8E309D0>
    └ <Handle Task.task_wakeup(<_GatheringFu...92522a6'}})]]>)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle Task.task_wakeup(<_GatheringFu...92522a6'}})]]>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle Task.task_wakeup(<_GatheringFu...92522a6'}})]]>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle Task.task_wakeup(<_GatheringFu...92522a6'}})]]>)>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
          └ <function _run_matcher at 0x000001E7EB1A9750>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _event_to_me at 0x000001E7EB0B6A70>: <Task finished name='Task-88834804' coro=<_event_to_me() done, defined at C:\...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x000001E78087EAA0>
          │       │   │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │       │   │    └ PrivateMessageEvent(time=1735149945, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │       │   └ Bot(type='OneBot V11', self_id='3973508735')
          │       └ <function Matcher.run at 0x000001E7EB0B68C0>
          └ Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _event_to_me at 0x000001E7EB0B6A70>: <Task finished name='Task-88834804' coro=<_event_to_me() done, defined at C:\...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x000001E78087EAA0>
          │    │          │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │    │          │    └ PrivateMessageEvent(time=1735149945, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │    │          └ Bot(type='OneBot V11', self_id='3973508735')
          │    └ <function Matcher.simple_run at 0x000001E7EB0B6830>
          └ Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 825, in simple_run
    await handler(
          └ Dependent(call=_)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
                 │    │             │         │    │    │       └ {'session': Session(bot_id='3973508735', bot_type='OneBot V11', platform='qq', level=<SessionLevel.LEVEL1: 1>, id1='274727782...
                 │    │             │         │    │    └ <function _ at 0x000001E7F586E5F0>
                 │    │             │         │    └ Dependent(call=_)
                 │    │             │         └ ~R
                 │    │             └ <class 'collections.abc.Awaitable'>
                 │    └ typing.Callable
                 └ <function cast at 0x000001E7E8C104C0>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\parse_bilibili\__init__.py", line 86, in _
    get_url = (qqdocurl_value if qqdocurl_value else jumpUrl_value).split("?")[
               │                 │                   └ {}
               │                 └ {}
               └ {}

AttributeError: 'dict' object has no attribute 'split'
12-26 02:05:46 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 02:05:46 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11)
12-26 02:05:56 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11) running complete
12-26 02:05:56 [ERROR] nonebot | Running Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11) failed.
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x000001E7EB056290>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x000001E7EB0560E0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000001E7EB82BD00>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 579, in run
    server.run()
    │      └ <function Server.run at 0x000001E7EB82B640>
    └ <uvicorn.server.Server object at 0x000001E7FC34C7C0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000001E7EB82B6D0>
           │       │   └ <uvicorn.server.Server object at 0x000001E7FC34C7C0>
           │       └ <function run at 0x000001E7E8D80E50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x000001E7EDB73DF0>
           │    └ <function BaseEventLoop.run_until_complete at 0x000001E7E8E80280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x000001E7EAA91510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x000001E7E8E81CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x000001E7E8E309D0>
    └ <Handle Task.task_wakeup(<_GatheringFu...result=[None]>)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle Task.task_wakeup(<_GatheringFu...result=[None]>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle Task.task_wakeup(<_GatheringFu...result=[None]>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle Task.task_wakeup(<_GatheringFu...result=[None]>)>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
          └ <function _run_matcher at 0x000001E7EB1A9750>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _event_to_me at 0x000001E7EB0B6A70>: <Task finished name='Task-88834804' coro=<_event_to_me() done, defined at C:\...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x000001E78087EAA0>
          │       │   │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │       │   │    └ PrivateMessageEvent(time=1735149945, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │       │   └ Bot(type='OneBot V11', self_id='3973508735')
          │       └ <function Matcher.run at 0x000001E7EB0B68C0>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _event_to_me at 0x000001E7EB0B6A70>: <Task finished name='Task-88834804' coro=<_event_to_me() done, defined at C:\...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x000001E78087EAA0>
          │    │          │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │    │          │    └ PrivateMessageEvent(time=1735149945, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │    │          └ Bot(type='OneBot V11', self_id='3973508735')
          │    └ <function Matcher.simple_run at 0x000001E7EB0B6830>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 825, in simple_run
    await handler(
          └ Dependent(call=_)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
                 │    │             │         │    │    │       └ {}
                 │    │             │         │    │    └ <function _ at 0x000001E7FACDEEF0>
                 │    │             │         │    └ Dependent(call=_)
                 │    │             │         └ ~R
                 │    │             └ <class 'collections.abc.Awaitable'>
                 │    └ typing.Callable
                 └ <function cast at 0x000001E7E8C104C0>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\nonebot_plugin_sparkapi\matchers\block_private.py", line 18, in _
    await matcher_blockprivate.finish("私聊功能已关闭！如有需要，请联系管理员。")
          │                    └ <classmethod(<function Matcher.finish at 0x000001E7EB0B5D80>)>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 582, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {}
          │   │    └ '私聊功能已关闭！如有需要，请联系管理员。'
          │   └ <classmethod(<function Matcher.send at 0x000001E7EB0B5CF0>)>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.block_private, lineno=11)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 566, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {}
                 │   │          │              └ '私聊功能已关闭！如有需要，请联系管理员。'
                 │   │          └ PrivateMessageEvent(time=1735149945, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
                 │   └ <function Bot.send at 0x000001E7EB5B4EE0>
                 └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\bot.py", line 237, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {}
                 │    │                      │     │      └ '私聊功能已关闭！如有需要，请联系管理员。'
                 │    │                      │     └ PrivateMessageEvent(time=1735149945, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
                 │    │                      └ Bot(type='OneBot V11', self_id='3973508735')
                 │    └ <attribute '__class__' of 'object' objects>
                 └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\bot.py", line 190, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2747277822, 'message_type': 'private', 'message': [MessageSegment(type='text', data={'text': '私聊功能已关闭！如有需要，请联系管理员...
                 └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\adapter\bot.py", line 122, in call_api
    raise exception
          └ ActionFailed(status='failed', retcode=1200, data=None, message='Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgServic...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\adapter\bot.py", line 97, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2747277822, 'message_type': 'private', 'message': [MessageSegment(type='text', data={'text': '私聊功能已关闭！如有需要，请联系管理员...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ Bot(type='OneBot V11', self_id='3973508735')
                   │    │       └ <function Adapter._call_api at 0x000001E7E8958820>
                   │    └ Adapter(name='OneBot V11')
                   └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\adapter.py", line 147, in _call_api
    return handle_api_result(await self._result_store.fetch(seq, timeout))
           │                       │    │             │     │    └ 30.0
           │                       │    │             │     └ 73018
           │                       │    │             └ <function ResultStore.fetch at 0x000001E7EB5B6050>
           │                       │    └ <nonebot.adapters.onebot.store.ResultStore object at 0x000001E7EB2AD960>
           │                       └ Adapter(name='OneBot V11')
           └ <function handle_api_result at 0x000001E7EB545000>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\utils.py", line 58, in handle_api_result
    raise ActionFailed(**result)
          │              └ {'status': 'failed', 'retcode': 1200, 'data': None, 'message': 'Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgServic...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: ActionFailed(status='failed', retcode=1200, data=None, message='Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{}\n\n    at Timeout.sendDataCallback [as _onTimeout] (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7666:15)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)', wording='Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{}\n\n    at Timeout.sendDataCallback [as _onTimeout] (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7666:15)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)', echo='73018')
12-26 03:01:00 [ERROR] zhenxun | 清理AI日常聊天数据失败... || 错误 <class 'FileNotFoundError'>: [WinError 3] 系统找不到指定的路径。: 'C:\\Users\\Administrator\\AppData\\Local\\nonebot2\\nonebot_plugin_sparkapi'
12-26 03:01:00 [INFO] zhenxun | 自动更新群组成员信息成功...
12-26 03:01:00 [INFO] zhenxun | 自动更新好友信息成功...
12-26 03:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 03:25:01 [INFO] zhenxun | CMD[自动备份] 自动备份成功...
12-26 03:25:01 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:01:00], next run at: 2024-12-26 03:26:00 CST)" was missed by 0:00:01.095070
12-26 04:01:00 [WARNING] zhenxun | 当前无UP池或 prts_up_char.json 文件不存在
12-26 04:01:01 [WARNING] zhenxun | guardian_up_char 解析出错
12-26 04:01:01 [WARNING] zhenxun | 赛马娘UP更新出错 || 错误<class 'AttributeError'>: 'NoneType' object has no attribute 'find'
12-26 04:01:01 [WARNING] zhenxun | pretty_up_char 解析出错
12-26 04:01:04 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者水
12-26 04:01:07 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者草
12-26 04:01:10 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者雷
12-26 04:01:12 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者岩
12-26 04:01:13 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者风
12-26 04:01:16 [INFO] zhenxun | 原神 更新成功
12-26 04:01:16 [INFO] zhenxun | 原神 武器更新成功
12-26 04:01:16 [INFO] zhenxun | 成功获取原神当前up信息...当前up池: 「陵薮市朝」 & 「渊海界令」 & 「神铸赋形」
12-26 04:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 04:30:07 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/09/08/00/01/37/111530368_p0.jpg 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\4.jpg
12-26 04:30:07 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/4.jpg --> /resources\image\_setu/4.jpg
12-26 04:30:07 [ERROR] zhenxun | 更新色图 4.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\4.jpg' -> 'resources\\image\\_setu\\4.jpg'
12-26 04:30:08 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2022/12/30/21/04/18/104052603_p0.jpg 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\3.jpg
12-26 04:30:08 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/3.jpg --> /resources\image\_setu/3.jpg
12-26 04:30:08 [ERROR] zhenxun | 更新色图 3.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\3.jpg' -> 'resources\\image\\_setu\\3.jpg'
12-26 04:30:12 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/09/09/18/30/20/111577264_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\2.jpg
12-26 04:30:23 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/01/01/17/17/43/104132686_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\1.jpg
12-26 04:30:25 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2022/10/13/01/09/34/101889280_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\0.jpg
12-26 04:30:26 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2022/01/28/15/00/34/95827678_p0.jpg 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\0.jpg
12-26 04:30:27 [INFO] zhenxun | CMD[自动更新色图] ("2024-12-26 04:30:27 更新 色图 完成，本地存在 6 张，实际更新 4 张，以下为更新时未知错误：\n更新色图 4.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\\\temp\\\\4.jpg' -> 'resources\\\\image\\\\_setu\\\\4.jpg'",)
12-26 05:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 06:01:00 [INFO] zhenxun | begin to build uni message!!!
12-26 06:01:00 [INFO] zhenxun | group list is [<GroupConsole>, <GroupConsole>, <GroupConsole>]
12-26 06:01:00 [INFO] zhenxun | key is 1003578395:None, check_func is <function check at 0x000001E7F4B4D630>
12-26 06:01:00 [INFO] zhenxun | is_run value False
12-26 06:01:00 [INFO] zhenxun | key is 992333365:None, check_func is <function check at 0x000001E7F4B4D630>
12-26 06:01:00 [INFO] zhenxun | is_run value True
12-26 06:01:00 [INFO] zhenxun | target is Target({'id': '992333365', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
12-26 06:01:00 [INFO] zhenxun | begin to build uni message!!!
12-26 06:01:01 [ERROR] zhenxun | CMD[被动早晚安] [Target]([992333365:None]) 发送失败 || 错误 <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>: ActionFailed(status='failed', retcode=1200, data=None, message='Error: EventChecker Failed: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{\n    "result": 120,\n    "errMsg": ""\n}\n\n    at file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7704:15', wording='Error: EventChecker Failed: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{\n    "result": 120,\n    "errMsg": ""\n}\n\n    at file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7704:15', echo='73493')
12-26 06:01:01 [INFO] zhenxun | key is 546881717:None, check_func is <function check at 0x000001E7F4B4D630>
12-26 06:01:01 [INFO] zhenxun | is_run value True
12-26 06:01:01 [INFO] zhenxun | target is Target({'id': '546881717', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
12-26 06:01:01 [INFO] zhenxun | begin to build uni message!!!
12-26 06:01:02 [INFO] zhenxun | 每日早安发送...
12-26 06:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 07:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 08:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 09:01:00 [INFO] zhenxun | begin to build uni message!!!
12-26 09:01:00 [INFO] zhenxun | group list is [<GroupConsole>, <GroupConsole>, <GroupConsole>]
12-26 09:01:00 [INFO] zhenxun | key is 1003578395:None, check_func is None
12-26 09:01:00 [INFO] zhenxun | is_run value True
12-26 09:01:00 [INFO] zhenxun | target is Target({'id': '1003578395', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
12-26 09:01:00 [INFO] zhenxun | begin to build uni message!!!
12-26 09:01:00 [INFO] zhenxun | key is 992333365:None, check_func is None
12-26 09:01:00 [INFO] zhenxun | is_run value True
12-26 09:01:00 [INFO] zhenxun | target is Target({'id': '992333365', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
12-26 09:01:00 [INFO] zhenxun | begin to build uni message!!!
12-26 09:01:01 [ERROR] zhenxun | CMD[被动每日新闻] [Target]([992333365:None]) 发送失败 || 错误 <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>: ActionFailed(status='failed', retcode=1200, data=None, message='Error: EventChecker Failed: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{\n    "result": 120,\n    "errMsg": ""\n}\n\n    at file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7704:15', wording='Error: EventChecker Failed: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{\n    "result": 120,\n    "errMsg": ""\n}\n\n    at file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7704:15', echo='73857')
12-26 09:01:01 [INFO] zhenxun | key is 546881717:None, check_func is None
12-26 09:01:01 [INFO] zhenxun | is_run value True
12-26 09:01:01 [INFO] zhenxun | target is Target({'id': '546881717', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
12-26 09:01:01 [INFO] zhenxun | begin to build uni message!!!
12-26 09:01:01 [INFO] zhenxun | 每日摸鱼发送...
12-26 09:01:01 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:01:01 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:01:02 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:01:02 [ERROR] zhenxun | 下载 ['https://api.03c3.cn/api/zb'] 下载超时.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\moyu1735174860.jpeg
12-26 09:01:02 [WARNING] zhenxun | 图片路径不存在: resources\temp\moyu1735174860.jpeg
12-26 09:01:02 [INFO] zhenxun | begin to build uni message!!!
12-26 09:01:02 [ERROR] apscheduler | Job "_ (trigger: cron[hour='9', minute='1'], next run at: 2024-12-27 09:01:00 CST)" raised an exception
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x000001E7EB056290>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x000001E7EB0560E0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000001E7EB82BD00>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 579, in run
    server.run()
    │      └ <function Server.run at 0x000001E7EB82B640>
    └ <uvicorn.server.Server object at 0x000001E7FC34C7C0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000001E7EB82B6D0>
           │       │   └ <uvicorn.server.Server object at 0x000001E7FC34C7C0>
           │       └ <function run at 0x000001E7E8D80E50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x000001E7EDB73DF0>
           │    └ <function BaseEventLoop.run_until_complete at 0x000001E7E8E80280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x000001E7EAA91510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x000001E7E8E81CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x000001E7E8E309D0>
    └ <Handle <TaskStepMethWrapper object at 0x000001E7806E6080>()>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x000001E7806E6080>()>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle <TaskStepMethWrapper object at 0x000001E7806E6080>()>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle <TaskStepMethWrapper object at 0x000001E7806E6080>()>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\apscheduler\executors\base_py3.py", line 30, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
                   │   │     │   │       │   └ <member 'kwargs' of 'Job' objects>
                   │   │     │   │       └ <Job (id=aea298b7c2c14c4f9a67d2c76a3584d9 name=_)>
                   │   │     │   └ <member 'args' of 'Job' objects>
                   │   │     └ <Job (id=aea298b7c2c14c4f9a67d2c76a3584d9 name=_)>
                   │   └ <member 'func' of 'Job' objects>
                   └ <Job (id=aea298b7c2c14c4f9a67d2c76a3584d9 name=_)>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\moyu.py", line 101, in _
    await broadcast_group(receipt, log_cmd="被动每日新闻")
          │               └ []
          └ <function broadcast_group at 0x000001E7EF7AAD40>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\utils\platform.py", line 593, in broadcast_group
    raise ValueError("群聊广播消息不能为空")

ValueError: 群聊广播消息不能为空
12-26 09:01:18 [INFO] zhenxun | 自动生成日报成功...
12-26 09:01:55 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
12-26 09:10:44 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 2062360669 from 1095712946@[群:546881717] '[at:qq=3973508735] 卡了？'
12-26 09:10:44 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:10:45 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:10:45 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:10:45 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:10:45 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:10:45 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:10:45 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
12-26 09:10:45 [INFO] zhenxun | 用户[1095712946] CMD[ai] 问题：卡了？ ---- 回答：None
12-26 09:10:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
12-26 09:40:25 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1013271108 from 843298567@[群:546881717] '[at:qq=3973508735] 摸鱼日历'
12-26 09:40:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:40:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:40:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:40:25 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:40:25 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:40:25 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:40:26 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.plugins.moyu, lineno=43)
12-26 09:40:26 [INFO] zhenxun | begin mo yu
12-26 09:40:26 [INFO] zhenxun | begin to build uni message!!!
12-26 09:40:26 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:40:26 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:40:26 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:40:26 [ERROR] zhenxun | 下载 ['https://api.03c3.cn/api/zb'] 下载超时.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\moyu1735177226.jpeg
12-26 09:40:26 [INFO] zhenxun | 每日图片下载路径 resources\temp\moyu1735177226.jpeg
12-26 09:40:26 [WARNING] zhenxun | 图片路径不存在: resources\temp\moyu1735177226.jpeg
12-26 09:40:26 [INFO] zhenxun | begin to build uni message!!!
12-26 09:40:26 [INFO] zhenxun | begin to build uni message!!!
12-26 09:40:27 [ERROR] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[843298567] CMD[moyu] 摸鱼接口错误 || 错误 <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>: ActionFailed(status='failed', retcode=1200, data=None, message='Error: 消息体无法解析, 请检查是否发送了不支持的消息类型\n    at OneBotMsgApi.sendMsgWithOb11UniqueId (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:26617:13)\n    at SendMsg._handle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33384:56)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async SendMsg.websocketHandle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33283:23)\n    at async OB11ActiveWebSocketAdapter.handleMessage (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:9469:21)', wording='Error: 消息体无法解析, 请检查是否发送了不支持的消息类型\n    at OneBotMsgApi.sendMsgWithOb11UniqueId (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:26617:13)\n    at SendMsg._handle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33384:56)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async SendMsg.websocketHandle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33283:23)\n    at async OB11ActiveWebSocketAdapter.handleMessage (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:9469:21)', echo='73945')
12-26 09:40:27 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.plugins.moyu, lineno=43) running complete
12-26 09:40:38 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.private.friend]: Message 2117146080 from 843298567 '摸鱼日历'
12-26 09:40:38 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:40:38 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:40:38 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:40:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:40:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:40:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:40:38 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.plugins.moyu, lineno=43)
12-26 09:40:38 [INFO] zhenxun | begin mo yu
12-26 09:40:38 [INFO] zhenxun | begin to build uni message!!!
12-26 09:40:38 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:40:38 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:40:38 [WARNING] zhenxun | 下载 https://api.03c3.cn/api/zb 失败.. 尝试下一个地址..
12-26 09:40:38 [ERROR] zhenxun | 下载 ['https://api.03c3.cn/api/zb'] 下载超时.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\moyu1735177238.jpeg
12-26 09:40:38 [INFO] zhenxun | 每日图片下载路径 resources\temp\moyu1735177238.jpeg
12-26 09:40:38 [WARNING] zhenxun | 图片路径不存在: resources\temp\moyu1735177238.jpeg
12-26 09:40:38 [INFO] zhenxun | begin to build uni message!!!
12-26 09:40:39 [INFO] zhenxun | begin to build uni message!!!
12-26 09:40:39 [ERROR] zhenxun | Adapter[OneBot V11] 平台[qq] 用户[843298567] CMD[moyu] 摸鱼接口错误 || 错误 <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>: ActionFailed(status='failed', retcode=1200, data=None, message='Error: 消息体无法解析, 请检查是否发送了不支持的消息类型\n    at OneBotMsgApi.sendMsgWithOb11UniqueId (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:26617:13)\n    at SendMsg._handle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33384:56)\n    at async SendMsg.websocketHandle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33283:23)\n    at async OB11ActiveWebSocketAdapter.handleMessage (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:9469:21)', wording='Error: 消息体无法解析, 请检查是否发送了不支持的消息类型\n    at OneBotMsgApi.sendMsgWithOb11UniqueId (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:26617:13)\n    at SendMsg._handle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33384:56)\n    at async SendMsg.websocketHandle (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:33283:23)\n    at async OB11ActiveWebSocketAdapter.handleMessage (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:9469:21)', echo='73948')
12-26 09:40:39 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.plugins.moyu, lineno=43) running complete
12-26 09:45:48 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 2475618 from 457891411@[群:546881717] '服务器被玩坏了...'
12-26 09:45:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:45:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:45:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:45:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:45:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:45:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:45:49 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
12-26 09:45:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
12-26 09:46:10 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 996087969 from 1095712946@[群:546881717] '服务器被玩坏了...'
12-26 09:46:10 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:46:10 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:46:10 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:46:10 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:46:10 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:46:10 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:46:10 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
12-26 09:46:10 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
12-26 09:46:34 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1771790880 from 2363080731@[群:546881717] '服务器被玩坏了...'
12-26 09:46:34 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:46:34 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:46:34 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:46:34 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:46:34 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:46:34 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:46:34 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
12-26 09:46:34 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
12-26 09:46:44 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1304528276 from 457891411@[群:546881717] '信不信哥哥用大光纤狠狠地接入你的服务器'
12-26 09:46:44 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:46:44 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:46:44 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:46:44 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:46:44 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:46:44 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:46:44 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
12-26 09:46:44 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
12-26 09:48:01 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1642877080 from 457891411@[群:546881717] '然后在你的数据库里恶意注入哥哥的代码'
12-26 09:48:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
12-26 09:48:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-26 09:48:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
12-26 09:48:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-26 09:48:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
12-26 09:48:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
12-26 09:48:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
12-26 09:48:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
12-26 09:59:12 [INFO] uvicorn | Shutting down
12-26 09:59:12 [WARNING] nonebot | OneBot V11 | WebSocket for Bot 3973508735 closed by peer
12-26 09:59:12 [INFO] websockets | connection closed
12-26 09:59:12 [INFO] uvicorn | Waiting for application shutdown.
12-26 09:59:12 [INFO] nonebot_plugin_apscheduler | Scheduler Shutdown
12-26 09:59:12 [INFO] nonebot_plugin_htmlrender | Browser Stopped.
12-26 09:59:12 [INFO] uvicorn | Application shutdown complete.
12-26 09:59:12 [INFO] uvicorn | Finished server process [3156]
